### PR TITLE
td#4

### PR DESCRIPTION
**Pull request by ezisberta**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+//     });
+// })
+
+describe("GET /api/categories", () => {
+  it("should respond with an array of category objects", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((res) => {
+        res.body.categories.forEach((category) => {
```
Comment (Rosem10) [inline __tests__/app.test.js:22]: Please also test the length of this return array or we will get a false positive if we are returned an empty array!
If we look through 'for each' of the items in an empty array, we will never fail our test because it will find nothing wrong.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  getReviewByID,
+} = require("./controllers/categories.controller");
+
+app.get("/api/categories", getCategories);
+
+app.get("/api/reviews/:review_id", getReviewByID);
+
+app.use((err, req, res, next) => {
+  console.log(err);
+  if (err.code === "22P02") {
```
Comment (Rosem10) [inline app.js:14]: Lovely error handling!

**Review by Rosem10 — CHANGES_REQUESTED** [review]
One tiiiiny change please!

**Review by Rosem10 — APPROVED** [review]
